addons/test_mail/tests/test_message_post.py

```diff
@@ -1001,6 +1001,68 @@ def test_message_post_w_attachments_on_main_attachment_model(self):
                         ]
         )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.addons.mail.models.mail_message_schedule')
+    def test_message_post_w_attachments_filtering(self):
+        """
+        Test the message_main_attachment heuristics with an emphasis on the XML/Octet/PDF types.
+        -> we don't want XML nor Octet-Stream files to be set as message_main_attachment
+        """
+        xml_attachment, octet_attachment, pdf_attachment = [('List1', b'My xml attachment')], [('List2', b'My octet-stream attachment')], [('List3', b'My pdf attachment')]
```

Comment:
:socks: Multiline a bit ? Try to stay < ~ 120 chars (not a hard constraint but here it becomes long :) )

Reply:
Was hesitating breaking those lines, will do